src/components/RegisterForm.vue

```diff
@@ -0,0 +1,276 @@
+<template>
+	<form @submit.prevent="handleSubmit(!v$.$invalid)" class="p-fluid">
+		<div class="field mb-4">
```

Comment:
Since we have so many identical inputs, it's much better to style them using sass in the `<style>` section, instead of manually setting margins using classes like this. We don't want to have to change all of those instances of `mb-4` if we decide to make the margins smaller or bigger.
Also, I would make the margins bigger, since everything gets a bit cramped when the inputs are filled:
![image](https://user-images.githubusercontent.com/51292024/160253838-750036f8-b394-470f-b12c-204f3bff669c.png)

Reply:
+1